docker/webui/run_openqa.sh

```diff
@@ -7,9 +7,36 @@ else
   /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
 fi
 
+function scheduler(){
+  su -u /usr/share/openqa/script/openqa-scheduler-daemon
+}
+
+function websockets(){
+  su geekotest -c /usr/share/openqa/script/openqa-websockets-daemon
+}
+
+function gru() {
+  su geekotest -c /usr/share/openqa/script/openqa-gru -m production run
+}
+
+function webui() {
+  su geekotest -c /usr/share/openqa/script/openqa-webui-daemon &
+  apache2ctl start
```

Comment:
Why are we using apache here? I thought we're using already haproxy in this setup? Two proxies seem to be quite redundant.

Reply:
I agree with that. My mission is to allow the port binding of the webui daemon to 0.0.0.0 to avoid the use of a in-container reverse proxy

Reply:
There should be a command line parameter which you simply need to set to change the listening address.

Reply:
an environment variable can suffice as well